promise-it-wont-hurt-solutions/src/02-fullfill-a-promise.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+var promise = new Promise(function(fulfill, reject) {
+  setTimeout(function() {
+    fulfill("FULFILLED!")
+  }, 300)
+});
```

Comment:
You can safely omit semicolons nearly everywhere (a few exceptions, like for loops)